modules/virt-configuring-storage-class-bootsource-update.adoc

```diff
@@ -11,7 +11,11 @@ You can override the default storage class by editing the `HyperConverged` custo
 
```

Comment:
In line 8, it says "custom" boot source updates, but it's relevant for both custom and system-defined boot sources, maybe it can be

`= Configuring a storage class for the boot source images
`

And we can remove 
`You can override the default storage class by editing the HyperConverged custom resource (CR).`
because in line 18 we say that one can specify the `storageClassName`